src/components/forms/spacing/spacing.tsx

```diff
@@ -78,7 +78,7 @@ export const Spacing = (props: SpacingProps) => {
           <Input
             type="number"
             suffix="px"
-            prefixIconSvg={IconSVG.BorderVertical}
+            prefixIconSvg={IconSVG.BorderHorizontal}
             defaultValue={currentSettings.top}
```

Comment:
```suggestion
            defaultValue={currentSettings.left}
```

But It's good I made the modification ! :)